cmd/migrate.go

```diff
@@ -21,10 +24,17 @@ func init() {
 }
 
 func migrateCmd(cmd *cobra.Command, args []string) {
-	c := config.LoadConfig()
+	var cfg *server.Config
+	loader := config.NewLoader(config.WithPath("./"))
+
+	if err := loader.Load(&cfg); err != nil {
+		fmt.Printf("ERROR: %s\n", err.Error())
+		os.Exit(1)
+	}
+
 	m, err := migrate.New(
-		c.DB.MigrationsPath,
-		c.DB.ConnectionString)
+		cfg.DB.MigrationsPath,
```

Comment:
I see we are copying migration files manually inside the docker image. Why not just embed them inside the binary?

Reply:
That's good idea. Yes we should embed them. Will add it backlog

Reply:
Ideally, I am thinking of not having migration files at all and only provide a migration guide in docs after a new release. WDYT?

Reply:
Didn't get you. Am I suppose to create db changes manually?

Reply:
no. migration guide provides the comamnds. But maybe we don't need to keep all migrations in service itself.